File: Web/RMA-Web-Shell/projects/shared-components-lib/src/lib/wizard/shared/models/start-wizard-request.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';
import { WizardPermissionOverride } from './wizard-permission-override';

export class StartWizardRequest extends BaseClass {
    type: string;
    linkedItemId: number;
    data: string;
    lockedToUser: string;
    customStatus: string;
    customRoutingRoleId: number;
    allowMultipleWizards: boolean;
    wizardPermissionOverrides: WizardPermissionOverride;
}

File: Service/ClientCare/RMA.Service.ClientCare.BusinessProcessTasks/InterBankTransfer/Model/StartWizardRequest.cs
namespace RMA.Service.ClientCare.BusinessProcessTasks.InterBankTransfer.Model
{
    public class StartWizardRequest
    {
        public string Type { get; set; }
        public int LinkedItemId { get; set; }
        public string Data { get; set; }
        public string LockedToUser { get; set; }
        public string CustomStatus { get; set; }
        public int? CustomRoutingRoleId { get; set; }
    }
}
